docs/guide/en/dropdown.md

```diff
@@ -0,0 +1,215 @@
+# Dropdown
+
+The **Dropdown** widget renders a Bootstrap 5 dropdown menu component within the **Yii framework**.
+
+You can use it to create interactive dropdown menus with various item types such as links, buttons, headers, dividers,
+or plain text. This component is highly customizable, supporting directions, alignments, themes, and toggle variations.
+
+## Key Features
+- Supports multiple item types: links, buttons, headers, dividers, and text.
+- Customizable toggle button or link with variants, sizes, and split options.
+- Configurable menu alignment and direction using Bootstrap 5 classes.
+- Auto close behavior control (for example, close on inside/outside clicks).
+- Flexible HTML attribute and CSS class management.
+
+## Quick Start
+To get started, instantiate the **Dropdown** widget and add items to it. Here is a basic example:
+
+```php
+use Yiisoft\Bootstrap5\Dropdown;
+use Yiisoft\Bootstrap5\DropdownItem;
+
+echo Dropdown::widget()
+    ->togglerContent('Toggle Dropdown')
+    ->items(
+        DropdownItem::link('Action', '#'),
+        DropdownItem::link('Another action', '#'),
+        DropdownItem::divider(),
+        DropdownItem::link('Separated link', '#')
+    )
+    ->render();
+```
+
+This generates a simple dropdown menu with a toggle button and three clickable items separated by a divider.
+
+## Basic Usage
+
+### Instantiation
+Create a new **Dropdown** instance using the `widget()` method:
+
+```php
+$dropdown = Dropdown::widget();
+```
+
+### Adding Items
+Use the `items()` method to define menu items. Each item is an instance of **DropdownItem**:
+
+```php
+$dropdown = Dropdown::widget()
+    ->items(
+        DropdownItem::link('Home', '/home'),
+        DropdownItem::header('Options'),
+        DropdownItem::link('Profile', '/profile'),
+        DropdownItem::divider(),
+        DropdownItem::text('Logged in as Admin')
+    );
+```
+
+### Rendering
+Render the dropdown as HTML using `render()` or cast it to a string:
+
+```php
+// Using render()
+$html = $dropdown->render();
+
+// Using __toString()
+$html = (string) $dropdown;
+```
```

Comment:
Need to mention that casting to string happens automatically in HTML templates, but something tells me we'll have to repeat it in each guide section, so probably worth moving to widget docs and linking to it from this guide.

Reply:
What happens if I open the document directly from `dropdown.md`, and i don't open the `README.md`? 

Users are generally like that; i prefer to repeat it so the user doesn't see it.